src/key_io.cpp

```diff
@@ -309,3 +311,33 @@ bool IsValidDestinationString(const std::string& str)
 {
     return IsValidDestinationString(str, Params());
 }
+
+std::pair<CPubKey, CPubKey> DecodeSilentData(const std::vector<unsigned char>& data)
```

Comment:
37b0ca1c3d753cc61e0debf8ea59160bdf4127ec `assert data.size() >= SILENT_PAYMENT_V0_DATA_SIZE` (or early return if you don't expect the caller to have already checked this)